CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 2020-08-21:
+
+- added executable (neptun.py) to run deus with AWI tsunami grid files
```

Comment:
Should be "neptunus" if we stay to latin versions or neptune for the english one :wink: